utility/src/main/java/org/oppia/android/util/parser/html/BUILD.bazel

```diff
@@ -0,0 +1,91 @@
+"""
+General-purpose utilities for parsing & rendering HTML within the app.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "html_parser",
+    srcs = [
+        "HtmlParser.kt",
+    ],
+    visibility = ["//app:__pkg__"],
```

Comment:
This makes sense to keep as API visibility.

Reply:
fixed